test/travis_commands.jl

```diff
@@ -1,8 +1,8 @@
-# Pkg.clone("ImageMagick")
-# Pkg.build("ImageMagick")
+Pkg.clone("ImageMagick")
```

Comment:
Change for ImageMagick here: https://github.com/JuliaPlots/Plots.jl/pull/904
For GR, StatPlots and PlotUtils we sometimes have to develop in concert and time releases, so I'd like to keep as is.